QEfficient/transformers/models/gptj/modeling_gptj.py

```diff
@@ -0,0 +1,434 @@
+# -----------------------------------------------------------------------------
+#
+# Copyright (c)  2023-2024 Qualcomm Innovation Center, Inc. All rights reserved.
```

Comment:
same

Reply:
will make these changes in all files in seperate PR.